packages/google-tag-manager/index.js

```diff
@@ -46,6 +48,19 @@ module.exports = async function nuxtTagManager(_options) {
     async: true
   })
 
+  // prepend google tag manager <noscript> fallback to <body>
+  this.options.head.noscript.push({
+    vmid: 'gtm-noscript',
+    innerHTML: `<iframe src="${(options.noscriptURL || '//www.googletagmanager.com/ns.html')}?${queryString}" height="0" width="0" style="display:none;visibility:hidden"></iframe>`,
+    pbody: true
+  })
+
+  // disables sanitazion for gtm noscript as we're using .innerHTML
+  this.options.head.__dangerouslyDisableSanitizersByTagID = Object.assign({}, this.options.head.__dangerouslyDisableSanitizersByTagID, {
+    'gtm-noscript': ['innerHTML']
```

Comment:
Couldn't you add the key directly with `this.options.head.__dangerouslyDisableSanitizersByTagID['gtm-noscript'] = ['innerHTML']`?